git-multimail/README

```diff
@@ -388,6 +394,18 @@ multimailhook.replyToRefchange
     - The value "none", in which case the Reply-To: field will be
       omitted.
 
+multimailhook.refFilterInclusionRegex
+multimailhook.refFilterExclusionRegex
+
+    Regular expressions that can be used to limit refs for which email
+    updates will be sent.  It is an error to specify both an inclusion
+    and an exclusion regex.  If an inclusion regex is specified, emails
+    will only be sent for refs which match this regex.  If an exclusion
+    regex is specified, emails will be sent for all refs except those
+    that match this regex (or that match a predefined regex specific
+    to the environment, such as "^refs/notes" for most environments and
+    "^refs/notes|^refs/changes" for the gerrit environment).
```

Comment:
Can we change this to "whitespace-separated list of regex"? The `|` works, but makes the expression less readable (it might get cryptic if we have several non-trivial regex separated by `|`).

Note that we don't need to do that now, since we can add the whitespace-separated list idea later without breaking compatibility.